Classes/ContextMenu/PasteReferenceItemProvider.php

```diff
@@ -26,17 +26,9 @@
 
 class PasteReferenceItemProvider extends RecordProvider
 {
-    /**
-     * @var array<non-empty-string, array<non-empty-string, mixed>>
-     */
-    protected $itemsConfiguration = [
```

Comment:
Originally, that resetted (replaced) the `$itemsConfiguration` defined in
`\TYPO3\CMS\Backend\ContextMenu\ItemProviders\RecordProvider` completly.

That is not done now. Within `addItems()` a new item is added and than the whole
stuff processed.

That is not the way how custom providers should be implemented, already before but with this change now getting worse.

Doing so, this class **must** have been modified to extend from
`\TYPO3\CMS\Backend\ContextMenu\ItemProviders\AbstractProvider`
and not from `RecordProvider`.

Reply:
Having this as class-property I got an error shown, therefore I moved it in the method.
The definition to include all parents is reverted in 188625fc